troposphere/static/js/components/images/MyImageRequestsPage.jsx

```diff
@@ -128,8 +128,8 @@ export default React.createClass({
                 endDateText = moment(request.get("end_date")).format("MMM D, YYYY h:mm:ss a");
             }
 
-            var newMachineId = !!request.get("new_machine") ? request.get("new_machine").uuid : "N/A";
-            var newMachineProvider = !!request.get("new_machine_provider") ? request.get("new_machine_provider").name : "Unknown";
+            var newMachineId = request.get("new_machine") ? request.get("new_machine").uuid : "N/A";
+            var newMachineProvider = request.get("new_machine_provider") ? request.get("new_machine_provider").name : "Unknown";
```

Comment:
Looks like we have a lot of occurrences of `!!`. I can't recall an actual use case. like if you wanted to compare some var `test` you might `!!test === true`, but I don't think we ever do that.